### DISCUSSION TITLE
T-Pot Future Platform Support

### DISCUSSION THREAD

**Discussion by t3chn0m4g3** [telekom-security/tpotce#1256] [Polls]
Currently T-Pot is only supported on Debian 11, which is and will be our distro of choice for the foreseeable future.

However, if there was a post-install or other support for Fedora or Ubuntu, would this something you would be interested in?
Or would you rather prefer a light version that just needs `docker` and `docker-compose` to spin up (ARM64, AMD64) T-Pot?

**Comment by ziehmon** [comment]
Absolutely in love with the idea of T-Pot light! :heart:

**Comment by t3chn0m4g3** [comment]
And the winner is ... `T-Pot light`.

**Reply by t3chn0m4g3** [reply]
T-Pot Light is starting to take form 😄 

![image](https://user-images.githubusercontent.com/4318452/235751306-7bbfc94f-15d4-456b-8ad1-0dc4eff64878.png)
![image](https://user-images.githubusercontent.com/4318452/235751349-8bb83222-b1b7-46ba-82fc-b12cc1fc3d61.png)

**Reply by t3chn0m4g3** [reply]
... and now Ubuntu 😁
![image](https://user-images.githubusercontent.com/4318452/235757818-45f14fbe-9cb7-4296-8af7-ef3c6659f27c.png)

**Reply by t3chn0m4g3** [reply]
And finally (for now)... Debian 😃

![image](https://user-images.githubusercontent.com/4318452/235762616-b7123b45-ef95-45eb-b5cf-6a574e77e01a.png)
